detections/application/m365_copilot_agentic_jailbreak_attack.yml

```diff
@@ -0,0 +1,56 @@
+name: M365 Copilot Agentic Jailbreak Attack
+id: e5c7b380-19da-42e9-9e53-0af4cd27aee3
+version: 1
+date: '2025-09-25'
+author: Rod Soto
+status: experimental
+type: TTP
```

Comment:
Lets add a data_sources key to this yaml and create appropriate files in the  data_source s directory